frontend/src/auth/AuthProvider.js

```diff
@@ -26,8 +26,11 @@ export default function AuthProvider ({children}) {
 
     const deleteVideo = (videoId) => deleteVideoApi(videoId, token)
```

Comment:
die deleteVideo methode würde ich nicht in den AuthConstext schreiben. denke da sollten nur Auth sachen rein. vielleicht machst du nen eigenen context dafür ?

Reply:
gute Idee